crates/bevy_pbr/src/light_probe/environment_map.rs

```diff
@@ -96,6 +101,34 @@ pub struct EnvironmentMapLight {
     ///
     /// See also <https://google.github.io/filament/Filament.html#lighting/imagebasedlights/iblunit>.
     pub intensity: f32,
+
+    /// The rotation of the environment light map
```

Comment:
Nit: Documenting how this rotation is applied would be useful to reduce trial-and-error here.